packages/gatsby/cache-dir/loader.js

```diff
@@ -26,7 +26,7 @@ if (process.env.NODE_ENV !== `production`) {
 // Cache for `cleanAndFindPath()`. In case `match-paths.json` is large
 const cleanAndFindPathCache = {}
 
-const findMatchPath = (matchPaths, trimmedPathname) => {
+const findMatchPath = trimmedPathname => {
```

Comment:
This doesn't seem to take into account `__BASE_PATH__` (which was checked earlier in https://github.com/gatsbyjs/gatsby/blob/daa2af3255131d94de065127b215068e82a4a412/packages/gatsby/cache-dir/production-app.js#L72)

Not sure if that's an issue

Reply:
Great catch, It is an issue. Fixed by stripping the BASE_PATH (like we do in loader.cleanAndFindPath).

Reply:
Lovely, thanks!